Dockerfile-builder

```diff
@@ -28,3 +20,8 @@ RUN npm install -g \
     jshint@2.7.0 \
     stylus@0.49.2 \
     uglify-js@2.4.13
+
+COPY ./requirements/compiled.txt /app/requirements/compiled.txt
+
+RUN pip install --build ./build --no-cache-dir --no-deps \
+    -r requirements/compiled.txt
```

Comment:
Another dockerfile for local dev. Almost certainly the requirements for dev will be more than what we need for prod deployment.